app/sitemap.ts

```diff
@@ -28,11 +29,13 @@ const sitemap = (): MetadataRoute.Sitemap => {
   const currentDate = new Date().toISOString();
 
   // This maps the URL routes into Sitemap entries
-  return [...dynamicRoutes, ...staticPaths].sort().map(route => ({
-    url: `${baseUrlAndPath}/${route}`,
-    lastModified: currentDate,
-    changeFrequency: 'always',
-  }));
+  return [...dynamicRoutes, ...staticPaths, ...EXTERNAL_LINKS_SITMAP]
```

Comment:
Can we assign `[...dynamicRoutes, ...staticPaths, ...EXTERNAL_LINKS_SITMAP]` to a constant?

Reply:
normally yes

Reply:
What do you mean by "normally yes?" 😅